website/client-old/js/controllers/authCtrl.js

```diff
@@ -11,6 +11,14 @@ angular.module('habitrpg')
 
       $scope.logout = function() {
         localStorage.clear();
+
+        // Adapted from http://www.quirksmode.org/js/cookies.html
+        var cookies = document.cookie.split(';');
+        for (var i = 0; i < cookies.length; i++) {
+          var cookieName = cookies[i].match(/([^=]+)(?=\=)/);
```

Comment:
If this line is meant to return only the name of the cookie, then it's not working as desired; it returns an array. You can see what's in it by putting `console.log(cookieName);` just after this line. `match` returns an array, so perhaps you want the first element of that array?